README.md

```diff
@@ -28,32 +31,80 @@ The decision to fork the original mender-mcu-client was made to:
 * **Provide Official Support**: Ensure that the project receives the necessary attention and
   resources from Northern.tech to meet the needs of the community and enterprise users.
 
+## Getting started
 
-## Get Started
-
-Since the project is under active development, we recommend watching the repository or checking back
-regularly for updates. Detailed documentation and usage instructions will be provided as the project
-progresses.
-
+For using `mender-mcu` for the first time, we strongly recommend you start with our [Zephyr reference application](https://github.com/mendersoftware/mender-mcu-integration).
 
-## Get started with Zephyr OS
-
-For using `mender-mcu` for the first time, we strongly recommend you start with our Zephyr reference
-application.
-
-It can be found in the repository
-[mender-mcu-integration](https://github.com/mendersoftware/mender-mcu-integration). The policy is
-that the reference application's `main` branch will follow the latest stable branch of `mender-mcu`
+The policy is that the reference application's `main` branch will follow the latest stable branch of `mender-mcu`
 (this repository).
 
-In the `mender-mcu-repository` you will find:
+In the `mender-mcu-integration` repository you will find:
 * A reference `west` workspace for Zephyr OS builds, including `CMake` and `KConfig` configurations.
 * Sample application for how to use `mender-mcu` APIs.
 * A list of boards that we have so far tested.
 
 We recommend starting there to understand how we have designed our solution and then coming back
 here for more in-depth information.
 
+### Compatibility
+| Zephyr OS version | Bootloader |
```

Comment:
I don't think the bootloader should be in this table. On the one hand, because we don't plan to integrate with any other bootloader, and in the other hand because it is an update module specific thing.

When introducing the update module (I haven't read so far, but let's assume there is a section for it) then we shall specify that "zephyr-image" update module the module installed by default that integrates with MCUboot